conf/messages.en

```diff
@@ -48,12 +48,33 @@ signedOut.guidance = We did not save your answers.
 unauthorised.title = You can’t access this service with this account
 unauthorised.heading = You can’t access this service with this account
 
+monthlyreturns.dateConfirmNilPayments.title = Confirm nil payments to subcontractors this month
+monthlyreturns.dateConfirmNilPayments.heading.h1 = Confirm nil payments to subcontractors this month
+monthlyreturns.dateConfirmNilPayments.heading.h2 = Enter the tax month and year you are reporting nil payments for
+monthlyreturns.dateConfirmNilPayments.paragraph = A nil return is to record that you have not made any payments to subcontractors in the Construction Industry for the month shown below.
+monthlyreturns.dateConfirmNilPayments.hint.h1 = Tax months end on the 5th of the month.
+monthlyreturns.dateConfirmNilPayments.hint.h2 = For example, 6 May 2025 5 June 2025
+monthlyreturns.dateConfirmNilPayments.hint.h3 = For example, 9 2024.
```

Comment:
monthlyreturns.dateConfirmNilPayments.hint.h1 = Tax months end on the 5th of the month.
monthlyreturns.dateConfirmNilPayments.hint.h2 = For example, 6 May 2025 5 June 2025
monthlyreturns.dateConfirmNilPayments.hint.h3 = For example, 9 2024.

Replace with a single hint:

monthlyreturns.dateConfirmNilPayments.hint.h1 = Tax months end on the 5th of the month, like 6 May. For example, 6 May 2025 to 5 June 2025.

Reply:
Changed.